src/Panel.php

```diff
@@ -60,7 +60,10 @@ public function getPanel()
 		$installed = $this->decode($lockFile);
 
 		if ($this->error === NULL) {
+			$required = array_filter($required);
+			$installed = array_filter($installed);
```

Comment:
Not needed. But $installed['packages-dev'] is NULL = key exists :/
Maybe better:
if(is_null($installed['packages-dev'])){
    $installed['packages-dev'] = [];
}